<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZintTestForm</class>
 <widget class="QWidget" name="ZintTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>523</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>601</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lbl_barcode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>barcode will be here</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/green_box.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_barhead">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
